sources/packages/backend/libs/test-utils/src/factories/application.ts

```diff
@@ -154,5 +192,8 @@ export async function saveFakeApplicationDisbursements(
     fakeOriginalAssessment,
   );
   savedApplication.currentAssessment = savedOriginalAssessment;
+  if (options?.applicationStatus === ApplicationStatus.Draft) {
```

Comment:
Do we really need to make it null at this point? If so, we need also to take care of the `savedApplication.currentAssessment` for instance.
If the problem is related to having an intitutionLocation associated with the offering and application location, we can also receive two locations on the relations options.